DataMonitoring/pressure_analysis.py

```diff
@@ -0,0 +1,163 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd
+from time import sleep
+import os
+from scipy import signal
+from scipy.signal import savgol_filter
+from scipy.signal import lfilter
+
+#first test
+waterflowData = pd.read_csv(os.getcwd() + "/data/WaterFlowTest_08_08_16_42")
+tankData = waterflowData['low2'].to_numpy()
+amountOfData = len(tankData)
+currPressureData = np.asarray([0] * amountOfData)
+
+#second test
+waterflowData2 = pd.read_csv(os.getcwd() + "/data/waterflow_test_full_9_5_16_16")
+tankData2 = waterflowData2['low1'].to_numpy()
+amountOfData2 = len(tankData2)
+currPressureData2 = np.asarray([0] * amountOfData2)
+
+#third test
+waterflowData3 = pd.read_csv(os.getcwd() + "/data/waterflow_test_full_9_5_14_31 (1)")
+tankData3 = waterflowData3['low1'].to_numpy()
+amountOfData3 = len(tankData3)
+currPressureData3 = np.asarray([0] * amountOfData3)
+
+#function for detecting peaks in data
+##example: detect2(tankData)
+def detect2(data):
```

Comment:
Can we rename this to `detectPeaks` Or is this actually detecting large changes? in which case we should call it `detectTransition` or something